src/characters/Character.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+
+import React from 'react';
+import {Card} from 'react-bootstrap';
+
+const CARD_TYPE = {
+    "Ambassador": "warning",
+    "Assassin": "dark",
+    "Contessa": "danger",
+    "Captain": "info",
+    "Duke": "secondary",
```

Comment:
we’ll probably need to add more properties later but this was a pretty nice hack for now